martinLi/Quiz/queue1.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * 
+ * Given an array of positive numbers and a positive number ‘k,’ 
```

Comment:
Don't forget to include an overview of your approach as well as a note about the time and space complexity of your solution.